### ISSUE TITLE
how to create custom dataloader for with labels of varying length?

### ISSUE THREAD

**Issue by Nisnab** [pytorch/tutorials#539]
My image labels are in .csv format and it has labels in varying order.
![Screenshot_2019-06-20 transferl(1)](https://user-images.githubusercontent.com/17276742/59912925-66b7f000-9417-11e9-95a3-1aca37c4d88b.png)

i have created custom dataset. My labels are in separate csv files and they have varying length.
I have tried making customdataset and then dataloader.
I want to do transfer learning with Resnet. Can anyonehelp, I am a beginner to both pytorch and deep learning.
![Screenshot_2019-06-20 transferl(2)](https://user-images.githubusercontent.com/17276742/59913219-0c6b5f00-9418-11e9-8ce8-84a669796c95.png)

![Screenshot_2019-06-21 transferl(4)](https://user-images.githubusercontent.com/17276742/59915757-b0a3d480-941d-11e9-94d5-91c726d8d334.png)

![Screenshot_2019-06-21 transferl(2)](https://user-images.githubusercontent.com/17276742/59914010-b8fa1080-9419-11e9-91fe-2a143de2351d.png)
Upto this step, Everything is fine, However, when i create dataloader,
it shows problem. Can anyone help? i am a beginner.
![Screenshot_2019-06-21 transferl(3)](https://user-images.githubusercontent.com/17276742/59915497-12b00a00-941d-11e9-9632-50cf0121e60e.png)
![Screenshot_2019-06-21 transferl(1)](https://user-images.githubusercontent.com/17276742/59915568-38d5aa00-941d-11e9-86fc-123257ce8ef8.png)

**Comment by holly1238** [comment]
Closing this as it is more than 2 years old. If you are still having trouble, feel free to reopen this or open a new issue. You can also post your question in our PyTorch discussion forum where you'll get a quick response from our skilled community members (https://discuss.pytorch.org/).
